preload/link-header-on-subresource-types.tentative.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<title>Makes sure that Link headers on subresources preload resources</title>
+<meta name="timeout" content="long">
+<script src="/common/utils.js"></script>
+<script src="/resources/testharness.js"></script>
+<script src="/resource-timing/resources/resource-loaders.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/preload/resources/preload_helper.js"></script>
+
+<script>
+    const resources = {
+        style: {href: '/preload/resources/dummy-preloads-subresource.css', type: 'text/css', loader: load.stylesheet},
```

Comment:
This tests the current Chromium/WebKit behavior, but not what we discussed in https://github.com/whatwg/fetch/pull/1409, right?

Reply:
Right, I created it to expose the current behavior. It's not ready for review

Reply:
It now tests what we've discussed